events/notifications.go

```diff
@@ -92,11 +98,68 @@ func (t *NotificationEventProperties) FromMap(m map[string]string) {
 	_ = json.Unmarshal(b, &t)
 }
 
+// labelsTemplate is a helper func to generate the template for displaying labels
+func labelsTemplate(field string) string {
+	return fmt.Sprintf("{{if %s}}### Labels: \n{{range $k, $v := %s}}**{{$k}}**: {{$v}} \n{{end}}{{end}}", field, field)
+}
+
+// defaultTitleAndBody returns the default title and body for notification
+// based on the given event.
+func defaultTitleAndBody(event string) (title string, body string) {
+	switch event {
+	case EventCheckPassed:
+		title = "Check {{.check.name}} has passed"
+		body = fmt.Sprintf("Canary: {{.canary.name}} \nAgent: {{.agent.name}} {{if .check_status.message}}\nMessage: {{.check_status.message}} {{end}}\n%s \n\n[Reference]({{.permalink}})", labelsTemplate(".check.labels"))
```

Comment:
Only specigy agent if is non zero - can we also use a multiline string instead of a very long line?